src/collectors/glusterfs/glusterfs.py

```diff
@@ -0,0 +1,134 @@
+# coding=utf-8
+
+"""
+The GlusterFSCollector currently only collects latency percentages
+from the GlusterFS storage system.
+
+version 0.3 beta
+
+Documentation for GlusterFS profiling:
+http://gluster.readthedocs.org/en/latest/Administrator%20Guide/Monitoring%20Workload/
+
+#### Dependencies
+
+ * glusterfs [https://www.gluster.org/]
+ * Profiling enabled: gluster volume profile <VOLNAME> start
+
+"""
+
+import diamond.collector
+import subprocess
+import sys
+try:
+    import xml.etree.cElementTree as ET
+except ImportError:
+    import xml.etree.ElementTree as ET
+
+metric_base = "glusterfs."
+target_volume = ''
+target_brick = ''
+
+
+class GlusterFSCollector(diamond.collector.Collector):
```

Comment:
Should extend from `ProcessCollector` instead, then you don't need to do your own `subprocess.Popen` below